tests/integration/olp-cpp-sdk-dataservice-read/VersionedLayerGetAggregatedDataTest.cpp

```diff
@@ -159,6 +160,8 @@ TEST_F(VersionedLayerGetAggregatedDataTest, ParentTileFarAway) {
   ASSERT_TRUE(client.RemoveFromCache(aggregated_parent));
 
   testing::Mock::VerifyAndClearExpectations(network_mock_.get());
+
+  olp::utils::Dir::remove(settings.disk_path_mutable.get());
```

Comment:
This is only done in case none of the ASSERT_TRUE/EQ above fail. Better is to add it to the SetUp()/TearDown() to make sure it is either removed after test ends, resp. before test runs.